ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -251,6 +251,7 @@ const Annotations = React.memo(({ variant }) => {
           trigger={
             <ButtonLink size={svType && 'big'}>
               {svType || mainTranscript.majorConsequence.replace(/_/g, ' ')}
+              {svType && `:${svTypeDetail}`}
```

Comment:
what if theres no `svTypeDetail`, like for simple DUPs or DELs

Reply:
Hm... It is not good.